hbase-thrift/src/main/java/org/apache/hadoop/hbase/thrift2/client/ThriftConnection.java

```diff
@@ -160,14 +160,16 @@ public int getConnectTimeout() {
 
     @Override
     public Pair<THBaseService.Client, TTransport> getClient() throws IOException {
-      TSocket sock = new TSocket(connection.getHost(), connection.getPort());
-      sock.setSocketTimeout(connection.getOperationTimeout());
-      sock.setConnectTimeout(connection.getConnectTimeout());
-      TTransport tTransport = sock;
-      if (connection.isFramed()) {
-        tTransport = new TFramedTransport(tTransport);
-      }
+      TTransport tTransport = null;
```

Comment:
Why change the style here? No big difference?

Reply:
In 0.14.0, TSocket constructor throws TTransportException, so guarded inside try-catch.